Tasks/ServiceFabricUpdateAppVersions/Update-ApplicationVersions.ps1

```diff
@@ -80,7 +83,7 @@ try {
     $logIndent = "".PadLeft(2)
     foreach ($serviceManifestImport in $newAppManifestXml.ApplicationManifest.ServiceManifestImport)
     {
-        $serviceVersion = Update-ServiceVersions -VersionSuffix $versionSuffix -ServiceName $serviceManifestImport.ServiceManifestRef.ServiceManifestName -NewPackageRoot $newAppPackagePath -OldPackageRoot $oldAppPackagePath -LogIndent $logIndent -UpdateAllVersions:$updateAllVersions -LogAllChanges:$logAllChanges
+        $serviceVersion = Update-ServiceVersions -VersionValue $versionValue -ServiceName $serviceManifestImport.ServiceManifestRef.ServiceManifestName -NewPackageRoot $newAppPackagePath -OldPackageRoot $oldAppPackagePath -LogIndent $logIndent -UpdateAllVersions:$updateAllVersions -LogAllChanges:$logAllChanges -ReplaceVersion:$replaceVersion 
```

Comment:
Why not construct the new version here and pass it to Update-PackageVersion and Update-Service version instead of passing both VersionValue and ReplaceVersion parameter?